zaza/openstack/charm_tests/trilio/tests.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python3
+
+# Copyright 2018 Canonical Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Collection of tests for vault."""
+
+import logging
+
+import zaza.openstack.charm_tests.test_utils as test_utils
+
+
+class TrilioBaseTest(test_utils.OpenStackBaseTest):
+    """Base test class for charms."""
+
+    conf_file = None
+
+    def test_restart_on_config_change(self):
+        """Checking restart happens on config change.
+
+        Change debug mode and assert that change propagates to the correct
+        file and that services are restarted as a result
+        """
+        # Expected default and alternate values
+        set_default = {"debug": "False"}
```

Comment:
Use `bool` values here, otherwise the config_change helper will not be able to detect if you are trying to set a configuration option to a value that is already set in the model.